token/sdk/vault/pps.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vault
+
+import (
+	"github.com/pkg/errors"
+)
+
+type PublicParamsStorage struct {
+	Provider *Provider
+}
+
+func (p *PublicParamsStorage) PublicParams(networkID string, channel string, namespace string) ([]byte, error) {
```

Comment:
could we do this with the tokendb?

Reply:
under the hood, the tokens db is doing that, yes. But the Driver API that needs an implementation of the `PublicParametersStorage` storage interface should not depend on the tokens db.